packages/@uppy/screen-capture/src/ScreenCapture.tsx

```diff
@@ -32,6 +32,14 @@ function getMediaDevices() {
 const SUPPORTED_IMAGE_TYPES = ['image/png', 'image/jpeg', 'image/webp'] as const
 type SupportedImageType = (typeof SUPPORTED_IMAGE_TYPES)[number]
 
+export type ScreenCaptureStatus =
```

Comment:
Can’t we just use a single state here, like `'captured'`, and rely on capturedScreenshotUrl and recordedVideo to determine whether a screenshot or a video was captured, since both can’t be truthy at the same time?